source/next.config.js

```diff
@@ -21,6 +22,7 @@ module.exports = withSass({
     region,
     userPoolWebClientId,
     userPoolId,
+    DEMOMODE
```

Comment:
Can we rename this to isROMode or similar since it is a flag?

Reply:
done,